### ISSUE TITLE
Buffer Issue - DataBC Replication Issue

### ISSUE THREAD

**Issue by emjohnst** [bcgov/BCHeritage#977]
Received note from DataBC two new sites are failing replication:

•	EaQl-79 (Saddle Mountain Lookout Trail)
•	EbQk-28 (Historical Hot Springs Trail)

Both sites were added by kml (converted from shapefile) as polylines but I am seem unable to buffer the linework to turn it into a polygon - thus the polyline are causing the DataBC replication failure. 

Steps to replicate:
1. Go to Site Record and Edit Spatial Location
2. Under site boundary, click "LineString" to select
3. Click Add Buffer Below
4. Enter 5 m buffer, Click Add Feature
5. Expected Result: buffer applied and return to site boundary
Actual Result: no change to site (see console error message attached)

![Image](https://github.com/bcgov/BCHeritage/assets/139929541/001a46ff-3581-4928-9786-9ad3797eaefc)

**Comment by bferguso** [comment]
The estimate is a bit loose as I would need to get further into the issue to come up with a resolution However this appears on the surface to be similar to the large search issue we previously encountered, so we may be able to implement a similar solution to that. The workaround might be to do the buffering in ArcGis and then import that as a polygon. That would require the large geometry fix that should be implement as part of the Fossil implementation.

**Comment by emjohnst** [comment]
@bferguso I'll try to implement the workaround first - and wait on the large geometry issue to be resolved to see if this is connected to that issue.

**Comment by emjohnst** [comment]
Fixed these sites, if issue happens again in future will reopen new ticket.
